bpf-gpl/calculate-flags

```diff
@@ -85,6 +85,11 @@ elif [[ "${filename}" =~ .*wep.* ]]; then
   # Workload endpoint; recognised by CALI_TC_HOST_EP bit being 0.
   ep_type="workload"
   args+=("-DCALI_LOG_PFX=CALICOLO")
+elif [[ "${filename}" =~ .*xdp.* ]]; then
+  # XDP, so host endpoint.
+  ((flags |= CALI_TC_HOST_EP))
```

Comment:
If this is the flags for compiling the XDP program, we probably need "CALI_XDP_PROG", since it's the flag to differentiate between TC and XDP in the parser. It's defined in the bpf.h as "#define CALI_XDP_PROG   (1<<6)"